plugins/modules/storage/pmem/pmem.py

```diff
@@ -248,7 +321,96 @@ def pmem_is_dcpmm_installed(self):
         command = ['show', '-system', '-capabilities']
         return self.pmem_run_ipmctl(command)
 
+    def pmem_get_region_align_size(self, region):
+        aligns = []
+        for rg in region:
+            if rg['align'] not in aligns:
+                aligns.append(rg['align'])
+
+        return aligns
+
+    def pmem_get_available_region_size(self, region):
+        available_size = []
+        for rg in region:
+            available_size.append(rg['available_size'])
+
+        return available_size
+
+    def pmem_get_available_region_type(self, region):
+        types = []
+        for rg in region:
+            if rg['type'] not in types:
+                types.append(rg['type'])
+
+        return types
+
     def pmem_argument_check(self):
+        def convert_to_bytes(size_str):
+            size = int(size_pattn.match(size_str).group(1))
```

Comment:
Why not simply use `module.human_to_bytes()` to convert strings to amounts of bytes?

Reply:
Thank you for letting me know about human_to_bytes(). It looks great! I have applied the change to use the API.